addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -218,7 +218,7 @@ const FontFamilyPickerUserValueWidget = SelectUserValueWidget.extend({
         if (this.googleLocalFonts.length) {
             const googleLocalFontsEls = fontEls.splice(-this.googleLocalFonts.length);
             googleLocalFontsEls.forEach((el, index) => {
-                $(el).append(renderToElement('website.delete_google_font_btn', {
+                $(el).append(renderToFragment('website.delete_google_font_btn', {
```

Comment:
If I am not mistaken, this could stay `renderToElement`. Indeed, in this case, the font is self hosted and there is only one root element (as the cloud icon is not displayed in this case).

Reply:
I changed that line for the sake of consistency, so that the template is always rendered the same way.

IMO it's better this way, there isn't any drawbacks, and it future proofs the code, as this line will always behave the same as the other line. So that if someone changes the template, we're not risking another bug like this one.

If you still want me to revert it, let me know and I will.